WoWPro/WoWPro.lua

```diff
@@ -573,7 +573,7 @@ function WoWPro:OnEnable()
         WoWPro:dbp("PlayerGetTimerunningSeasonID function is not available")
     end
 
-    if _G.C_Seasons.GetActiveSeason() then
+    if WoWPro.CLASSIC and _G.C_Seasons.GetActiveSeason() then
```

Comment:
How about:
`if  _G.C_Seasons and _G.C_Seasons.GetActiveSeason() then`
instead?